db/migrate/20170320195445_create_books.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBooks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :books do |t|
+      t.string :title, null: false
+      t.string :author, null: false
+
+      t.timestamps
+    end
+    add_index :books, :title, unique: true
```

Comment:
I think you should add a newline above this. Usually if you have a block you can add a newline between it and the nextline